examples/gnrc_border_router/Makefile.native.conf

```diff
@@ -1,16 +1,25 @@
+SHELL:=/bin/bash
+
+TAP = tapbr0
+
 # native has a TAP interface towards the host, just add parameters for
 # `socket_zep`
 ZEP_PORT_BASE ?= 17754
-ZEP_PORT_MAX := $(shell expr $(ZEP_PORT_BASE) + $(ZEP_DEVICES) - 1)
+ZEP_PORT_MAX := $(shell echo $$(($(ZEP_PORT_BASE) + 2 * $(ZEP_DEVICES) - 1)))
 
 CFLAGS += -DSOCKET_ZEP_MAX=$(ZEP_DEVICES)
 CFLAGS += -DASYNC_READ_NUMOF=$(shell expr $(ZEP_DEVICES) + 1)
 
 # Set CFLAGS if not being set via Kconfig
 CFLAGS += $(if $(CONFIG_KCONFIG_MODULE_DHCPV6),,-DCONFIG_DHCPV6_CLIENT_PFX_LEASE_MAX=$(ZEP_DEVICES))
 
-# -z [::1]:$PORT for each ZEP device
-TERMFLAGS += $(patsubst %,-z [::1]:%, $(shell seq $(ZEP_PORT_BASE) $(ZEP_PORT_MAX)))
+# generate -z [::1]:$LOCAL_PORT,[::1]:$REMOTE_PORT argument for each ZEP device, starting with $ZEP_PORT_BASE
+# $REMOTE_PORT will be $LOCAL_PORT + 1
```

Comment:
Does this really make the usage of ZEP easier? As I read the comment it is now impossible to create a pair of nodes that are reachable to each other, which I'd say is the main use case.

Reply:
I'll change the PR and drop 40d4a6fd4e9a1f9802c04a778c0a0c2d8a685e67, modify this to use a UDP dispatcher instead.

This allows to connect multiple clients and is easier to implement than putting the logic into native / RIOT.

I'll squash since this basically re-writes the PR.